### PR TITLE
Add documentation for model cropbox.

### PR DESCRIPTION
**Pull request by anaritaflp**
This PR adds documentation for the model cropbox feature to the Teach page.

There's a new section `Editing the model` with two subsections: one about changing the pick frame and the other about cropping and expanding the model cloud.

I also removed section `Isolating the object`, as now this is not necessary anymore, since we can change the model afterwards with the cropbox.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -51,9 +51,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.10'
+version = '2.0'
 # The full version, including alpha/beta/rc tags.
-release = '1.10'
+release = '2.0'
```
Comment (viktorku) [inline docs/conf.py]: This feature is for v2.1, do we distinguish the minor versions yet?

File: docs/documentation/teach/how-to-use-pickit-teach.rst [hunk 2]
```diff
@@ -66,15 +57,55 @@ successfully defined, the viewer will open the **Model
...
+**Cropping and expanding the model**
+
+When clicking :guilabel:`Add a model`, the resulting model corresponds to
+the content of the current :ref:`region-of-interest`. However, the model
+point cloud can be cropped or expanded afterwards, using the arrows in the
+**Model** view.
+
+On the top left corner of the **Model** view, there is a **Auto-snap crop box**
+checkbox. If this option is enabled, the green model box automatically bounds
+the model points. This is useful for the user to check if the model contains
```
Comment (viktorku) [inline docs/documentation/teach/how-to-use-pickit-teach.rst]: Suggestion to rephrase into `If this option is enabled, the green model box automatically adjusts to the limits of the model points.`

File: docs/documentation/teach/how-to-use-pickit-teach.rst [hunk 3]
```diff
@@ -66,15 +57,55 @@ successfully defined, the viewer will open the **Model
...
+undesired points that are difficult to spot.
+
+**If there are no undesired points in the model, the auto-snapped model box bounds the model points cleanly.**
+.. image:: /assets/images/Documentation/Teach-model-autosnap-on-ghostpoints.png
+
+**If the model contains undesired points that are difficult to see, the auto-snapped model box seems larger than the actual model.**
+.. image:: /assets/images/Documentation/Teach-model-autosnap-on-clean.png
+
+If the auto-snap option is disabled, the green model box is whatever the user
```
Comment (viktorku) [inline docs/documentation/teach/how-to-use-pickit-teach.rst]: ...the green model box is **set to** whatever the user...

File: docs/documentation/teach/how-to-use-pickit-teach.rst [hunk 4]
```diff
@@ -66,15 +57,55 @@ successfully defined, the viewer will open the **Model
...
+defines using the model arrows.
+
+.. image:: /assets/images/Documentation/Teach-model-autosnap-off.png
+
+Notice that cropping or expanding the model does not replace the model with
```
Comment (viktorku) [inline docs/documentation/teach/how-to-use-pickit-teach.rst]: nit: `Note` instead of `Notice` sounds better to me

Comment (viktorku) [reply docs/documentation/teach/how-to-use-pickit-teach.rst]: Also the whole block can be styled in some kind of info box..

File: docs/documentation/teach/how-to-use-pickit-teach.rst [hunk 5]
```diff
@@ -66,15 +57,55 @@ successfully defined, the viewer will open the **Model
...
+the content of the region-of-interest, but only changes the point cloud that
+was previously taught.
```
Comment (viktorku) [inline docs/documentation/teach/how-to-use-pickit-teach.rst]: You can also mention something like:

<pre>
Even though the initial taught model contains only the points of the region-of-interest
at the time of teaching, you can tweak the model box to include points that were outside of it.
These points are static - they don't represent the live view of the camera,
but they are the ones that were grabbed when the model was taught.
</pre>

Perhaps @adolfo-rt can chime in for an even better phrasing here..

Comment (anaritaflp) [reply docs/documentation/teach/how-to-use-pickit-teach.rst]: @viktorku Thanks for the review! I changed this paragraph to include your suggestion, PTAL.

**Comment by PhilColpaert** [comment]
Personally I would prefer the following: 
Lets keep the article How to use Pickit Teach clean. Just the basics and more of the ideas how it works and what you can do. And create a different article, something like: How to edit/finetune my model after teaching it. And in this article you can go in depth with a good example.

**Pull request merged**